src/tools/web/client/src/components/TreeView.jsx

```diff
@@ -0,0 +1,72 @@
+// adapted from https://github.com/chenglou/react-treeview
```

Comment:
@markus2330 at the moment, I added input fields to tree elements with values (this would not be possible with the original library). As this will be the focus in further iterations of elektra-web, the tree view component will still be expanded with more functionality later. I also added comments about this in the file.